discovery-provider/src/models/aggregate_interval_play.py

```diff
@@ -13,9 +13,8 @@ class AggregateIntervalPlay(Base):
     track_id = Column(Integer, nullable=False, primary_key=True)
     genre = Column(Integer, nullable=False)
     created_at = Column(Integer, nullable=False)
-    week_listen_counts = Column(Integer, nullable=False)
-    month_listen_counts = Column(Integer, nullable=False)
-    year_listen_counts = Column(Integer, nullable=False)
+    week_listen_counts = Column(Integer, nullable=False, index=True)
+    month_listen_counts = Column(Integer, nullable=False, index=True)
```

Comment:
Is there a reason for the index?

Reply:
I think it might have been for the legacy system where we took the most played tracks and then scored those tracks for trending

Reply:
Yeah I just took it from the original mat view migration but not sure why it'd be used. I'll take it out.
https://github.com/AudiusProject/audius-protocol/blob/212b1aee26b1ce0b3fa78af6d49fdd7646585819/discovery-provider/alembic/versions/92571f94989a_add_trending_views.py#L73